src/main/java/edu/wpi/cs3733/D22/teamC/controller/service_request/medical_equipment/MedicalEquipmentSRCreateController.java

```diff
@@ -91,11 +90,13 @@ protected MedicalEquipmentSR clickSubmit(ActionEvent event) {
         System.out.println(requestIDString);
 
         //Dealing with the equipment type and the enumerator
-        int type = medEquip.getEquipEnum(equipType.getValue());
+        int type = medEquip.getEquipmentType().ordinal();
```

Comment:
Found the code we needed to store enums as numbers....

![image](https://user-images.githubusercontent.com/51765298/161402748-832ee034-e93d-4872-a4ce-557fce85d1d2.png)

Enum to int: **.ordinal()**
Int to enum: **Enumeration.values()[x]**

Maybe not the most important thing to do rn, but something to keep in mind when we move to refactoring.